catalogue/webapp/test/components/WorkItemStatus.test.js

```diff
@@ -0,0 +1,48 @@
+// @flow
+import WorkItemStatus from '@weco/catalogue/components/WorkItemStatus/WorkItemStatus';
+import { mountWithTheme } from '@weco/common/test/fixtures/enzyme-helpers';
+// import mockCatalogueWork from '@weco/catalogue/__mocks__/catalogue-work';
+import mockStacksWork from '@weco/catalogue/__mocks__/stacks-work';
+
+const availableItem = mockStacksWork.items.find(
+  item => item.status.label === 'Available'
+);
+const temporarilyUnavailableItem = mockStacksWork.items.find(
+  item => item.status.label === 'On holdshelf'
+);
+const unrequestableItem = mockStacksWork.items.find(
+  item => item.status.label === 'Missing'
+);
+
+describe('Feature: 1. As a library member I want to know if an item is available for requesting', () => {
+  test(`
+    Scenario: I'm viewing a work page
+      Given the work has a requestable item
+      And it's available to request
+      Then I can see that it's requestable
+  `, () => {
+    const wrapper = mountWithTheme(<WorkItemStatus item={availableItem} />);
+    expect(wrapper.find('[data-test-id="itemStatus"]')).toMatchSnapshot();
```

Comment:
I am battling to see the logic in when we use snapshots and when we don't?

Reply:
We were trying both ways of testing - but will settle on one